src/pages/service.tsx

```diff
@@ -59,10 +66,18 @@ const Service = ({ operator, passengerType, services }: ServiceProps): ReactElem
     </Layout>
 );
 
-export const getServerSideProps = async (ctx: NextPageContext): Promise<{}> => {
-    deleteCookieOnServerSide(ctx, SERVICE_COOKIE);
-
+export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props: ServiceProps }> => {
     const cookies = parseCookies(ctx);
+    const serviceCookie = cookies[SERVICE_COOKIE];
```

Comment:
Do we want the cookie deleted server side still? We could extract the cookies and then use the `deleteCookieOnServerSide` method?

Reply:
I'm not sure what purpose it serves by deleting it - on some pages its needed for the logic (the way it is written anyway) but on this page I dont think it needs to be deleted. Can you think of a scenario we should?